G1ANT.Addon.Images/G1ANT.Addon.Images/Api/Imaging.cs

```diff
@@ -46,97 +51,114 @@ public static bool AreImagesEqual(Bitmap image1, Bitmap image2)
 
         public static Rectangle IsImageInImage(Bitmap smallBmp, Bitmap bigBmp, double tolerance)
```

Comment:
I know this is not your code, but on the 26 line there's unused, deprecated method, remove it for code clarity